src/state.rs

```diff
@@ -7,5 +7,7 @@ use crate::operations::SwapOperationsList;
 /// As an MVP we hardcode paths for each tuple of assets (offer, ask).
 /// In a future version we want to find the path that produces the highest
 /// number of ask assets, but this will take some time to implement.
-pub const PATHS: Map<(AssetInfoKey, AssetInfoKey), SwapOperationsList> = Map::new("paths");
+/// To support multiple paths between the same asset, we add an id field we increment per asset of the same
+/// path
+pub const PATHS: Map<(AssetInfoKey, AssetInfoKey, u64), SwapOperationsList> = Map::new("paths");
```

Comment:
What about using `Map<..., Vec<SwapOperationsList>>`? I.e. if we want to find the best path, then we anyway need all of the possible paths, so I don't see an advantage in giving them separate keys.